acl/src/test/java/org/springframework/security/acls/domain/AclAuthorizationStrategyImplTests.java

```diff
@@ -41,15 +43,12 @@
 	@Mock
 	Acl acl;
 
-	GrantedAuthority authority;
+	GrantedAuthority authority = new SimpleGrantedAuthority("ROLE_SYSTEM_ADMIN");
 
 	AclAuthorizationStrategyImpl strategy;
 
-	@Before
-	public void setup() {
```

Comment:
Please revert this change and preserve the `@Before` method and create a `withUserRoles()` static helper method  instead.

Reply:
switching back to `@Before` method I realized that helper method is not needed